README.md

```diff
@@ -40,17 +40,10 @@ To get a local copy up and running follow these simple example steps.
 
```

Comment:
- [ ] Please remove the unused sections in the Readme from line 17 to line 38, because it helps your Readme to be more professional. you can add sections whenever you need them. what do you think?